<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image : url(:/images/background1.png)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>441</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid #00264A;
border-radius:8px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="Ok_pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>290</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #00264A;
color:white;</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
color:#00264A;
padding:4px;</string>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhoneNumber_lineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>230</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
color:#00264A;
padding:4px;</string>
     </property>
     <property name="placeholderText">
      <string>Phone Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Brush Script MT</family>
       <pointsize>40</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#00264A;
border:none;
background:none;</string>
     </property>
     <property name="text">
      <string>Blue Market</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="SignUp_radioButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#00264A;
border:none;
background:none;</string>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="SignIn_radioButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#00264A;
border:none;
background:none;</string>
     </property>
     <property name="text">
      <string>Sign in</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
